### ISSUE TITLE
What about 3.0 ?

### ISSUE THREAD

**Issue by Xety** [markstory/acl_extras#42]
Hello Mark,

Did you have the project to make the 3.0 version of your plugin ? It would be awesome ! Because actually, there is no plugin/script to extract all controllers/actions for the CakePHP3/ACL plugin.

I have started to convert it, but to be honest, it's a little hard, because somes fonctions has been removed from 2.x and there is no fonctions in Cake3 who can do the sames things. Exemple :
`App::uses`
`App::objects`

Regards.

**Comment by markstory** [comment]
I have no plans to update this plugin. If anything, the code could/should be merged with cakephp/app instead.

**Comment by rchavik** [comment]
Perhaps you meant `cakephp/acl` mark?

**Comment by markstory** [comment]
Yes, I did mean cakephp/acl @rchavik. I don't think I'll have time to move the code over until 3.0 stable unfortunately as I've been pretty busy with 3.0.

**Comment by aristos32** [comment]
hi guys. It's been a while you discussed this and I'm wondering what happened finally. I have the same issue, as I try to update my project from 2.10 to 3.0.

@rchavik did you manage to upgrade it?
@markstory do you suggest some alternative for 3.0? I reached a point where $menus variable is undefined in my views and I suspect is because I'm missing this plugin. 

Excuse me if I'm missing something, since I didn't implement it in 2.10 myself.

```
// Process Toolbar Buttons
foreach ($menus['pages_menu'] as $item) {

```

**Comment by markstory** [comment]
> do you suggest some alternative for 3.0?

I don't have any alternatives, and don't plan on dedicating time to this project. You're welcome to fork it and improve the compatibility.
